request_verification_api.py

```diff
@@ -0,0 +1,32 @@
+import requests
```

Comment:
Rename this file to api.py and create AuthenteqAPI class that makes requests to our API. It should have two methods: request_verification and retrieve_details. In constructor it should get CLIENT_ID and CLIENT_SECRET. It should be instantiated in the main file of the app.